code/index.html

```diff
@@ -2,21 +2,111 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width-device-width, initial-scale-1.0" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="flex-parent">
+      <div class="flex-child">
+        <h1>Daily Sports News</h1>
+      </div>
+      <div class="flex-logo">
+        <img src="News_logo.png" />
+      </div>
+      <div class="flex-child1">
+       <nav class="navigation-bar">
+          <a href="">Ice Hockey</a> 
+          <a href="">Football</a> 
+          <a href="">Winter Sports</a> 
+          <a href="">Other Sports</a> 
+        </nav>
+      </div>
     </header>
-
     <section class="big-news">
-      <!-- The big news content -->
+      <div class="big-new-img">
+        <img class="shadow-1" src="icehockey-news-site.jpg" alt="Ice hockey goalie">
+      </div>
+      <div class="big-news-text">
+        <h2>Season finally starts!</h2>
+        <p>
+          After months of eager waiting, NHL season has finally started.
+          <br>
+          Wednesday, the first games of the years were played in United States
+          of America.
+          <br>
+          Worth mentioning to an Swedish audience, is that several of the
+          players
+          <br>
+          from Sweden had successful evenings.
+          <br>
+          <br>
+          <br>
+          <a href="">Continue to article...</a>
+        </p>
+      </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <section class="grid-container">
+      <div class="grid-item">
+        <img class="shadow" src="football_arena.jpg" alt="Football arena">
+        <br>
+        <br>
```

Comment:
The `<br>` element is not semantic and we should try to avoid it. You can achieve the same result using margin in this case.